marlowe/src/Language/Marlowe/Pretty.hs

```diff
@@ -0,0 +1,79 @@
+{-# LANGUAGE DefaultSignatures #-}
+{-# LANGUAGE FlexibleContexts  #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE TypeOperators     #-}
+module Language.Marlowe.Pretty (prettify, Pretty, pretty) where
+
+import           GHC.Generics            ((:*:) ((:*:)), (:+:) (L1, R1), C, Constructor, D, Generic, K1 (K1), M1 (M1),
+                                          Rep, S, U1, conName, from)
+import           Text.PrettyPrint.Leijen (Doc, comma, encloseSep, hang, lbracket, line, parens, rbracket, space, text)
+
+appendWithLine :: Doc -> Doc -> Doc
+appendWithLine x y = x <> line <> y
```

Comment:
This is actually: https://www.stackage.org/haddock/lts-13.2/ansi-wl-pprint-0.6.8.2/Text-PrettyPrint-ANSI-Leijen.html#v:-60--36--62-

Reply:
yeah, I wanted a text version of it originally as it was going to be in a fold but now I could use the infix version except that it's very annoying that I will have to have `import Prelude hiding ((<$>))`

Reply:
Changed